Source/SwipeController.swift

```diff
@@ -382,7 +384,24 @@ extension SwipeController: UIGestureRecognizerDelegate {
             let view = gestureRecognizer.view,
             let gestureRecognizer = gestureRecognizer as? UIPanGestureRecognizer {
             let translation = gestureRecognizer.translation(in: view)
-            return abs(translation.y) <= abs(translation.x)
+
+            let location = gestureRecognizer.location(in: view)
+            guard let swipeable = swipeable else { return false }
+
+            var leftPanArea: CGFloat = 0
+            let width = swipeable.bounds.width
+            if let leftOptions = delegate?.swipeController(self, editActionsOptionsForSwipeableFor: .left) {
+                leftPanArea = width * leftOptions.leftSwipeAreaMultiplier
```

Comment:
We could add an option to provide either multiplier or absolute value, with an enum like
```
enum PanAreaWidth {
    case proportional(_ multiplier: CGFloat)
    case absolute(_ width: CGFloat)
}
```
This would make it easier to match whatever requirements we have for pan area